spec/javascripts/modules/contact-embed-modal-spec.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-env jasmine */
+/* global ContactEmbedModal, fetchMock, buildModalDialogue, removeModalDialogue */
+
+describe('ContactEmbedModal', function () {
+  'use strict'
+
+  var link, contactEmbedModal
```

Comment:
Sure, I can understand the confusion. I'm struggling a bit on a good name though. I want to try avoid button since the element is not a `<button>` and I feel `openButton` implies that. I tried to see if we had a naming already in the widget code that I could borrow but it's called "link" in HTML and "module" in JS.

Things that come to mind are: openControl, openTrigger, modalOpener.

Of those I probably like modalOpener the most so I'll do a quick change to that in an extra commit. Let me know your thoughts or any revised ideas.

Reply:
I thought the same and the fact that we already have links as buttons made the distinction seem somewhat irrelevant. I think `openLink` would be a good compromise - not so keen on `modalOpener`, I'm afraid, as it sounds like something I'd use for baked beans 😆.

Reply:
Sure, I can live with that.